src/main/java/pl/betoncraft/betonquest/config/Config.java

```diff
@@ -110,24 +114,42 @@ public Config(final boolean verboose) {
      * Creates package with the given name and populates it with default quest
      *
      * @param packName name of the new package
+     * @param populate if the files should be populated with the example quest or left empty
      * @return true if the package was created, false if it already existed
      */
     @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
-    public static boolean createDefaultPackage(final String packName) {
+    public static boolean createDefaultPackage(final String packName, final boolean populate) {
```

Comment:
This method doesn't create the default package anymore, so please refactor it to make the name reflect what it is doing.
```suggestion
    public static boolean createDefaultPackage(final String packName) {
        createPackage(packName, true);
    }

    public static boolean createPackage(final String packName, final boolean populate) {
```

Reply:
I'm unsure if this change does destroy some other code due to the renamed method name. I'll commit it when I have time to see what breaks.

Reply:
It does, but doing it like this will not break APIs.

This will break:
https://github.com/BetonQuest/BetonQuest/blob/18af936c9ffc95d3d591a17f068c616440919acf/src/main/java/pl/betoncraft/betonquest/commands/QuestCommand.java#L461
https://github.com/BetonQuest/BetonQuest/blob/18af936c9ffc95d3d591a17f068c616440919acf/src/main/java/pl/betoncraft/betonquest/config/Config.java#L87